src/widget/form/profileform.cpp

```diff
@@ -487,6 +487,7 @@ void ProfileForm::onRegisterButtonClicked()
     QString response = Toxme::createAddress(code, server, id, name, privacy, bio);
 
     Core* newCore = Core::getInstance();
+    // It is necessary if the user will logout (or logout and login) before request is finished
```

Comment:
I'll help you out with the comment a bit: "Make sure the user didn't logout (or logout and login) before the request is finished, else qTox will crash."